Backend/ChatAPI.Data/ChatAppDbContext.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.EntityFrameworkCore.Query.Internal;
+
+namespace ChatAPI.Data;
+using Microsoft.EntityFrameworkCore;
+
+public class ChatAppDbContext : DbContext
+{
+
+    protected override void OnConfiguring(DbContextOptionsBuilder options)
+    {
+        
+    }
+    
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        
+    }
+
+    public ChatAppDbContext(DbContextOptions<ChatAppDbContext> options) : base(options)
```

Comment:
Add constructors before the all other methods. 

Method order should be like as below in a class;

1. Constructor
2. Public Methods
3. Private Methods